src/test/cpp/util/file_windows_test.cc

```diff
@@ -34,6 +34,9 @@
 #error("This test should only be run on Windows")
 #endif  // !defined(_WIN32) && !defined(__CYGWIN__)
 
+#define _T(x) #x
```

Comment:
These two macros look quite odd to me. I also don't see why you want to use __LINE__ as a way to distinguish between targets. You could just give a random number, no? Is there any point to use the line number from the file?

If you do need this, I don't think you need to split it into two definitions, the single one should be enough.

Reply:
>  I also don't see why you want to use \_\_LINE\_\_ as a way to distinguish between targets.

Because it is guaranteed to be unique among all other tests. For a while there was no cleanup code that deleted temp files that tests created. Now there is, at least in some tests (the test framework won't do it automatically), but I still use this practice to avoid tests clobbering each other's files.

> If you do need this, I don't think you need to split it into two definitions, the single one should be enough.

Indeed! Done.